### ISSUE TITLE
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder"

### ISSUE THREAD

**Issue by mibby** [Realizedd/TokenManager#22]
Paper dev 1523 (Spigot 1.12.2)
TokenManager v3.1.3

```
[07:00:30] [Server thread/INFO]: [TokenManager] Enabling TokenManager v3.1.3
[07:00:30] [Server thread/INFO]: [TokenManager] Loaded Config.
[07:00:30] [Server thread/INFO]: [TokenManager] Loaded Lang.
[07:00:30] [Server thread/INFO]: [TokenManager] Loaded ShopsConfig.
[07:00:30] [Server thread/WARN]: SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
[07:00:30] [Server thread/WARN]: SLF4J: Defaulting to no-operation (NOP) logger implementation
[07:00:30] [Server thread/WARN]: SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
[07:00:30] [Server thread/INFO]: [TokenManager] Loaded DataManager.
[07:00:30] [Server thread/INFO]: [TokenManager] Loaded ShopListener.
[07:00:30] [Server thread/INFO]: [TokenManager] Successfully hooked into 'PlaceholderAPI'!
[07:00:30] [Server thread/INFO]: [TokenManager] Successfully hooked into 'Vault'!
[07:00:30] [Server thread/INFO]: [TokenManager] Loaded HookManager.
```

**Comment by Realizedd** [comment]
Will be fixed in the next update, thanks.

**Comment by mibby** [comment]
I believe ChestShop use to output this too and was subsequently fixed after I reported it if you want to see as a reference how they resolved the issue.

**Comment by Realizedd** [comment]
I already have fixed it. :) But I wanted to implement more features before releasing an update.

**Comment by mibby** [comment]
Bump. :(

Is there any other method you could use to resolve the issue? Or perhaps you could shade in the class?
https://www.slf4j.org/codes.html#StaticLoggerBinder

**Comment by mibby** [comment]
@Realizedd This is how Plan fixed the issue.
https://github.com/Rsl1122/Plan-PlayerAnalytics/pull/757


https://github.com/Rsl1122/Plan-PlayerAnalytics/pull/757/commits/2d328e1e3179c5c0093abdb0bd297110a92c53a8
https://github.com/Rsl1122/Plan-PlayerAnalytics/commit/05bb441b2dd8565f6f180231f9f8562cb89ef034

**Comment by Realizedd** [comment]
Fixed in c0acc20
